.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
Gatsby has nice extra links that appear at the bottom of the issues creation page:

![image](https://user-images.githubusercontent.com/60357436/84675553-52bca680-af24-11ea-8156-0f0c2db1dcc1.png)

It would be cool to update this with the docs site once that comes along. I'm not sure the Wiki is worth linking to directly with the current offerings. However we could potentially have a high level links for Termux? Slack? Common Windows questions?

Reply:
yea we for sure could/should I spent a bit too long searching through looking for things we can link to but there was nothing that really made sense, definitely something for the future